_squirtle/mod.ts

```diff
@@ -257,25 +302,24 @@ const runCheck = async (argv: any) => {
   const { ENV } = env
   const isDev = ENV !== 'prod'
   const siteUrl = isDev ? 'https://quest-dev.stellar.org' : 'https://quest.stellar.org'
-  
+
   if (!user.pk) {
     const missingPkConfirmed = await Confirm.prompt(`You have not yet connected your Stellar wallet. 
    This will affect your ability to claim NFT and XLM rewards.
    Would you like to connect your Stellar wallet?`);
 
     if (missingPkConfirmed) {
-      const run1 = Deno.run({
-        cmd: ['gp', 'preview', '--external', `${siteUrl}/settings`]
-      })
-      return run1.status()
+      return gp({
+        cmd: ['preview', '--external', `${siteUrl}/settings`]
+      }).catch(() => { console.warn('❌ This only works inside a gitpod') })
```

Comment:
Again why not use `browse` and why the catch? `sq` is meant for Gitpod only I don't want to pepper non-gitpod related errors everywhere.